wagn/lib/wagn/commands.rb

```diff
@@ -114,15 +122,16 @@ def find_spec_file filename, base_dir
     end
     parser.on('-k', '--decko-spec FILENAME(:LINE)', 'Run spec for a Wagn deck file') do |file|
       opts[:files] = find_spec_file( file, Decko.gem_root)
-      warn "files #{file}, #{Decko.gem_root}, #{opts[:files].inspect}"
     end
     parser.on('-c', '--core-spec FILENAME(:LINE)', 'Run spec for a Wagn core file') do |file|
       opts[:files] = find_spec_file( file, Cardio.gem_root)
     end
     parser.on('-m', '--mod MODNAME', 'Run all specs for a mod or matching a mod') do |file|
-      mod_path = "#{Cardio.gem_root}/mod/#{file}"
-      if File.exists? mod_path
+      if File.exists? mod_path = "mod/#{file}"
+        opts[:files] = "#{Cardio.gem_root}/mod/#{file}"
+      elsif File.exists? mod_path = "#{Cardio.gem_root}/mod/#{file}"
         opts[:files] = "#{Cardio.gem_root}/mod/#{file}"
+      elsif (opts[:files] = find_spec_file( file, "mod")).present?
```

Comment:
Note, this is the original case of finding them in the local app root dir.  I kind of like also using -m for gem modules, but maybe they should have the lists merged instead of as a fallback.